### PR TITLE
UM-040 - Chaplain Hidden Gun (Faith)

### PR DESCRIPTION
**Pull request by UrsulaMejor**
## About the PR 

- Adds Faith, the chaplain's personal sidearm
- Faith is a four shot .22 Beretta that can be hidden in the chaplain's personal bible.
- Only the chaplain has enough faith to remove their sidearm from their personal bible.

## Why's this needed? 

'Cause ya gotta have Faith

Joking aside, this is just to give the chaplain something cool and personal besides their bible healing powers, and is part of the larger work we've been doin to give the average crew member some more options to fight with without having to go out of their way to grab them from debris and such, especially with the more limited access to stun weaponry we've pushed.

## Changelog

```
(u)UrsulaMajor and makkipakki:
(*)The chaplain now starts with a four shot .22 Beretta in their personal bible. Sprites by makkipakki
```

### CODE REVIEW

**Comment by UrsulaMejor** [comment]
Video:
https://cdn.discordapp.com/attachments/585526776550391819/728827856892854282/2020-07-03_21-20-30.mp4

**Comment by UrsulaMejor** [comment]
@tarm please move silencing to be a var on the gun instead of being on the ammo :^)

File: code/obj/item/storage/bible.dm [hunk 1]
```diff
@@ -225,3 +226,32 @@ var/list/bible_contents = list()
...
+		..()
+
+	attackby(var/obj/item/W, var/mob/user)
+		if(istype(W,/obj/item/gun/kinetic/faith))
+			if (user.traitHolder && user.traitHolder.hasTrait("training_chaplain"))
+				user.u_equip(W)
+				W.set_loc(src)
+				user.show_text("You hide [W] in \the [src].", "blue")
+				return
+		..()
```
Comment (MarkNais) [inline code/obj/item/storage/bible.dm:257]: I assume a chaplain can still open a "loaded" bible without pulling faith by dragging the bible onto their sprite.

Comment (UrsulaMejor) [reply code/obj/item/storage/bible.dm:257]: They can, yeah

**Pull request merged**